deploy.sh

```diff
@@ -28,7 +28,14 @@ Variables:
 These variables have default values defined in the script. The defaults can be
 overridden by environment variables. Any environment variables are overridden
 by values set in a '.env' file (if it exists), and in turn by those set in a
-file specified by the '--config-file' option."
+file specified by the '--config-file' option.
+
+Positional Args:
+
+At the end of the command, you can optionally specify the directory, branch,
+and repository as well. Earlier values are required to specify later ones. For
+example, in order to specify <branch>, you must also specify <directory>. Like
+the command-line options, these"
```

Comment:
Last minute coding before travel while tired didn't serve me here. b91b54e completes that sentence.